eng/Testing.targets

```diff
@@ -57,6 +65,9 @@
 
     <!-- Otherwise, run tests on AzDO CI agents -->
     <SkipTests Condition=" '$(IsAzdoCIRunner)' == 'true' and '$(RunOnAzdoCI)' == 'true' ">false</SkipTests>
+
+    <!-- If we're running locally, we don't skip tests -->
+    <SkipTests Condition=" '$(IsLocalDevLoop)' == 'true' ">false</SkipTests>
```

Comment:
Locally we don't skip any tests (unless those were explicitly skipped).
For example, there are some tests which are declared as incompatible with Windows because there's no Docker on Windows build agents. Docker can be installed locally, meaning that such test can be run locally as well.